app/views/miq_request/_prov_vm_grid.html.haml

```diff
@@ -85,8 +86,9 @@
             = h(number_to_human_size(@vm.mem_cpu.to_i * 1024 * 1024))
           %td
             = h(number_to_human_size(@vm.allocated_disk_storage))
-          %td
-            = h(@vm.deprecated ? _("true") : _("false"))
+          - if @edit[:vm_headers].key? 'deprecated'
```

Comment:
You should omit parentheses only when you're using a DSL and here you're clearly not.